### PR TITLE
Unify the name in [k8s / kubernetes] and [storage class / StorageClass]  …

### PR DESCRIPTION
**Pull request by daniel-hutao**
Unify the name in [k8s / kubernetes] and [storage class / StorageClass]  …

Signed-off-by: daniel-hutao <farmer.hutao@outlook.com>

### CODE REVIEW

File: roles/ks-core/config/tasks/main.yaml [hunk 1]
```diff
@@ -55,16 +55,16 @@
     {{ bin_dir }}/kubectl get secret kubesphere-secret -o jsonpath='{.data.secret}' | base64 -d
   register: ks_secret_str
 
-- name: KubeSphere | Check k8s version
+- name: KubeSphere | Checking kubernetes version
```
Comment (LinuxSuRen) [inline roles/ks-core/config/tasks/main.yaml:58]: `kubernetes`

Please capitalize the first letter.

File: roles/ks-core/config/tasks/main.yaml [hunk 2]
```diff
@@ -55,16 +55,16 @@
...
   shell: >
     {{ bin_dir }}/kubectl version -o json | jq '.serverVersion.gitVersion' | sed s/\"//g
   register: kubernetes_version
 
 - debug:
-    msg: Current Kubernetes version is {{kubernetes_version.stdout}}
+    msg: Current kubernetes version is {{kubernetes_version.stdout}}
```
Comment (LinuxSuRen) [inline roles/ks-core/config/tasks/main.yaml:64]: `kubernetes`

Please capitalize the first letter.

Kubernetes or KubeSphere are all special word instead the normal one.

File: roles/ks-core/ks-core/tasks/main.yaml [hunk 3]
```diff
@@ -125,7 +125,7 @@
   when:
     - ks_svc.stdout.find("ks-console") != -1
 
-- name: KubeSphere | import ks-core status
+- name: KubeSphere | Import ks-core status
```
Comment (LinuxSuRen) [inline roles/ks-core/ks-core/tasks/main.yaml:128]: Why don't use `importing` here?

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]

File: roles/ks-core/ks-core/tasks/main.yaml (hunk 3 above)
Comment (daniel-hutao) [reply roles/ks-core/ks-core/tasks/main.yaml:128]: If it is needed, I would do change it to importing later.

File: roles/ks-core/config/tasks/main.yaml (hunk 1 above)
Comment (daniel-hutao) [reply roles/ks-core/config/tasks/main.yaml:58]: #1256

File: roles/ks-core/config/tasks/main.yaml (hunk 2 above)
Comment (daniel-hutao) [reply roles/ks-core/config/tasks/main.yaml:64]: #1256

File: roles/ks-core/ks-core/tasks/main.yaml (hunk 3 above)
Comment (daniel-hutao) [reply roles/ks-core/ks-core/tasks/main.yaml:128]: #1256

**Pull request merged**